html-css/week1/yasi/order-desktop/css/main.css

```diff
@@ -0,0 +1,313 @@
+/* BODY */
+body {
+  font-family: Arial, sans-serif;
+  background: rgba(35, 16, 22, 0.3);
+  color: #333;
+  line-height: 1.6;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+/* HEADER */
+.header {
+  text-align: center;
+  width: 100%;
+  max-width: 500px;
+  padding: 20px;
+  background-color: rgba(255, 255, 255, 0);
+  color: white;
+  margin-top: 30px;
+  margin-bottom: 20px;
+}
+
+.header h1 {
+  font-size: 2rem;
+  margin: 0;
+  font-weight: bold;
+}
+
+.header p {
+  font-size: 1rem;
+  margin: 5px 0 0;
+}
+
+/* MAIN CONTAINER */
+.main-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 500px;
+  background: none; /* Transparent background */
+  padding: 0; /* No padding between sections */
+  margin: 0; /* No extra margin around container */
+  box-sizing: border-box;
+}
+
+/* ORDER SUMMARY */
+.order-summary {
+  background: linear-gradient(92.74deg, #fd7b46 1.15%, #fe4e9b 98.48%);
+  color: white;
+  padding: 20px;
+  text-align: left;
+  border-radius: 8px;
```

Comment:
This also sets radius to the bottom corners. To target only the top 2, you would have to set it as:
```suggestion
  border-radius: 8px 8px 0 0;
```
This first sets the top left corner, then top right and so on in clock-wise direction.
![border-radius](https://github.com/user-attachments/assets/90170bac-85df-45f6-bb5e-1deee2ec02f0)